sdks/java/core/src/main/java/org/apache/beam/sdk/worker/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Utilities for configuring worker environment. */
+package org.apache.beam.sdk.worker;
```

Comment:
@lukecwik do you have any objection to dropping this in the top-level of sdk core and calling it something like `BeamJvmInitializer` instead?

Reply:
I wouldn't stick this in `org.apache.beam.sdk` because there is purposefully very little there (`Pipeline`, `PipelineResult`, ...)

If the term `worker` is the issue, then I would suggest using `harness` and put this in `org.apache.beam.sdk.harness` and name the class `BeamHarnessInitializer`.

Reply:
Ok I renamed the `worker` package to `harness` and `BeamWorkerInitializer` to `JvmInitializer` (I figured "beam" and "harness" were both redundant, while "JVM" communicates that this is meant to initialize any JVMs that are started).

So now the new interface and class are `org.apache.beam.sdk.harness.JvmInitializer` and `org.apache.beam.fn.harness.JvmInitializers`.